install/linux.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_label: Linux
+---
+
+# Installing RGBDS on Linux
+
+## Using your package manager
+
+First, please check if RGBDS is available in your distribution's software center or package manager repositories.
+If a satisfactory version is available, it is a good idea to install from there.
+
+## Using our pre-built binaries
+
+:::info Availability
+
+This is only available for RGBDS v0.6.1 or newer.
+
+:::
+
+1. Go the [latest release](https://github.com/gbdev/rgbds/releases/latest) (or pick a specific version from [the list](https://github.com/gbdev/rgbds/releases))
```

Comment:
To make this easier for people to skim by looking at links and headings or other emphasized text:
```suggestion
1. Go the [latest release](https://github.com/gbdev/rgbds/releases/latest) (or pick [a specific version](https://github.com/gbdev/rgbds/releases))
```